src/platform/i386/kernel.c

```diff
@@ -67,14 +69,19 @@ kern_memory_setup(struct multiboot *mb, u32_t mboot_magic)
 	for (i = 0 ; i < mb->mods_count ; i++) {
 		struct multiboot_mod_list *mod = &mods[i];
 
-		printk("\t- %d: [%08x, %08x)", i, mod->mod_start, mod->mod_end);
+		char *cmdline = (char *)mod->cmdline;
+		int cmdline_len = strnlen((const char*)cmdline, CMDLINE_MAX_LEN);
+		assert(cmdline_len >= CMDLINE_REQ_LEN);
```

Comment:
Assertion should be _after_ the definitions.  (don't mix definitions and statements)